dimagi/pages/templates/pages/about.html

```diff
@@ -1,63 +1,60 @@
-{% extends "base.html" %}
-{% load i18n %}
-{% load dimagidotcom %}
-
-{% block title %}{% blocktrans %}
-  About Dimagi
-{% endblocktrans %}{% endblock %}
-
-{% block description %}{% blocktrans %}
-  Our technology, CommCare, the world’s most powerful mobile data
-  collection platform, was built with this goal in mind. Using
-  CommCare, organizations build mobile apps that improve the
-  quality and efficiency of services in the most remote corners
-  of the world.
-{% endblocktrans %}{% endblock %}
-
-{% block body_class %}about-page tucked-hero{% endblock %}
-
-{% block stylesheets %}
-  {% compress css %}
-    <link href="{% sass_src 'style/pages/about.scss' %}" rel="stylesheet" media="screen" type="text/css" />
-  {% endcompress %}
-{% endblock %}
-
-{% block hero %}
- <div id="top"></div>
-  <section class="hero hero-about"
-           data-lazybg="true"
-           id="hero">
-    <div class="content align-left-at-medium align-center">
-      <h2 class="gap-vert-line-at-medium gap-top-large">
-        {% blocktrans %}
-          About Dimagi
-        {% endblocktrans %}
-      </h2>
-      <h1>
-        {% blocktrans %}
-          Empowering People to Make an Impact
-          and Help Those in Need
-        {% endblocktrans %}
-      </h1>
-    </div> {# .content #}
-  </section>
-{% endblock %}
-
-{% block content %}
-  <section class="about-container orange-theme">
-    {% include "sections/about/sub_nav.html" %}
-    {% include "sections/about/mission.html" %}
-    {% include "sections/about/resources.html" %}
-    {% include "sections/about/team.html" %}
-    {% include "sections/about/management.html" %}
-    {% include "sections/about/company_info.html" %}
-    {% include "sections/contact/offices.html" %}
-    {% include "sections/about/contact.html" %}
-  </section>
-{% endblock %}
-
-{% block modals %}
-  {% for employee in management %}
-    {% include "partials/team_modal.html" with employee=employee %}
-  {% endfor %}
-{% endblock %}
+{% extends "base.html" %}

```

Comment:
why is this diff showing all the lines have changed?

Reply:
May be due to the whitespaces it showing like that even thought it is not highlighting them. But, If u select the **hide white space changes like below** then it is showing the changes properly
![screenshot from 2019-02-12 12-03-08](https://user-images.githubusercontent.com/47313114/52616330-3b981f80-2ebe-11e9-8f49-325e20713258.png)

![screenshot from 2019-02-12 12-04-45](https://user-images.githubusercontent.com/47313114/52616415-726e3580-2ebe-11e9-88f1-6e4a06f6f2ea.png)